citrination_client/client.py

```diff
@@ -10,10 +10,13 @@
 client and method_name are constant (avoides reassignment
 in the assignment loop in the client class)
 """
+
+
 def _generate_lambda_proxy_method(client, method_name):
```

Comment:
you probably want to remove this method if you're removing the dynamic method definitions